src/Elastic.Apm/AgentComponents.cs

```diff
@@ -72,6 +72,13 @@ public class AgentComponents : IApmAgent, IDisposable
 				}
 				else
 					Logger.Info()?.Log("The Elastic APM .NET Agent is disabled - the agent won't capture traces and metrics.");
+
+				TracerInternal = new Tracer(Logger, Service, PayloadSender, ConfigurationStore,
+					currentExecutionSegmentsContainer ?? new CurrentExecutionSegmentsContainer(), ApmServerInfo, breakdownMetricsProvider);
+
+#if NET5_0
+				new OpenTelemetry.ElasticActivityListener(this, TracerInternal);
+#endif
```

Comment:
Since the Elastic.Apm takes a dependency on System.Diagnostics.DiagnosticSource 5.0.0, can the listener be enabled for all supported TFMs?